plugins/woocommerce-admin/client/products/product-form.tsx

```diff
@@ -84,6 +85,8 @@ export const ProductForm: React.FC< {
 				) }
 			</ProductFormLayout>
 			<ProductFormFooter />
+			{ /* @ts-expect-error 'scope' does exist. @types/wordpress__plugins is outdated. */ }
+			<PluginArea scope="woocommerce-product-editor" />
```

Comment:
I think having this specifically here does seem like the best approach.
Although in relation to my musings [here](https://github.com/woocommerce/woocommerce/pull/36368/files#r1066696332).

I wonder if we should actually just clone a similar version of `PluginArea` call it `ProductEditorArea` and remove the use of slot fills and make use of this instead?
It feels slightly tedious to me, that we use this first (which acts very similar to a SlotFill) and then use SlotFills within that to put things in the right location.
Anyway me just thinking out loud.

Reply:
Overall I think I like this, although I'm not entirely clear on the thought since your "here" link is actually broken. Does this sound like something that could be added as a later optimization?

Reply:
Weird the `here` link is broken, it's suppose to reference the other comment I left above with musings. 
Yeah this could be added as a later optimization, I think more something that will be determined by how things are progressing when more plugins update and see where annoyances lie.